docs/source/development/quickstart.rst

```diff
@@ -62,32 +62,32 @@ for further details.
 
 .. literalinclude:: ../../../examples/pj_obs_api_mini_demo.c
   :language: c
-  :lines: 56
+  :lines: 57
   :dedent: 4
 
-The coordinate defined above is transformed with ``proj_trans_coord``. For this
+The coordinate defined above is transformed with ``proj_trans``. For this
```

Comment:
Another question: I could replace things like ``` ``proj_trans`` ``` in this file with `` :c:func:`proj_trans` `` so it automatically creates a link to the reference page. Would there be interest in a PR for that?

Reply:
Yes, definitely. There are many of such occurrences. I usually fix them when I stumble upon them. Any help is greatly appreciated.

Reply:
OK, then I will do a PR for that. Perfect combination while reading through the docs to directly do some fixes :)

Reply:
Thanks. I'm sure there's many things that can be improved. It's been a long time since I've read the docs from start to finish so I've lost track of how consistent it is. All improvements are welcome :)